### PR TITLE
Add Marp

### PR DESCRIPTION
**Pull request by caius**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/marp.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'marp' do
+  version '0.0.8'
+  sha256 '6db7310232bf9abb4629f5a1600ebb9b912ffd750aa44eeaaa7c7ebdf5f563c6'
+
+  # github.com was verified as official when first introduced to the cask
+  url 'https://github.com/yhatt/marp/releases/download/v0.0.8/0.0.8-Marp-darwin-x64.dmg'
```
Comment (vitorgalvao) [inline Casks/marp.rb]: `url "https://github.com/yhatt/marp/releases/download/v#{version}/#{version}-Marp-darwin-x64.dmg"`

Comment (vitorgalvao) [inline Casks/marp.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/marp.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'marp' do
+  version '0.0.8'
+  sha256 '6db7310232bf9abb4629f5a1600ebb9b912ffd750aa44eeaaa7c7ebdf5f563c6'
+
+  # github.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/marp.rb]: Comment need to be more specific (`github.com/yhatt/marp`).

**Comment by caius** [comment]
@vitorgalvao thanks for the feedback, I think I've addressed it all. 😄

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**